docker-compose.yaml

```diff
@@ -16,18 +16,32 @@ services:
     container_name: nginx
     build:
       context: .
-      dockerfile: Dockerfile.nginx   
+      dockerfile: Dockerfile.nginx
     depends_on:
-      - pythonapp
+      - certbot   
     networks:
       app-network:
         ipv4_address: 172.25.0.11
     volumes:
       - ./logs:/var/log/nginx
-    ports: 
-      - 80:80
-
-
+      - ./certs:/etc/letsencrypt
```

Comment:
![image](https://github.com/OpsStrike/dos14-Labzeev_Timofey-git-flow/assets/34745780/eb525865-df10-47c2-8818-b6424c4392bc)
I have this error after second attempt running `docker-compose up`